src/Parser/Mage/Fire/Modules/Features/Cinderstorm.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import SpellIcon from 'common/SpellIcon';
+import Wrapper from 'common/Wrapper';
+import { formatNumber } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import Analyzer from 'Parser/Core/Analyzer';
+import AbilityTracker from 'Parser/Core/Modules/AbilityTracker';
+
+class Cinderstorm extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+    abilityTracker: AbilityTracker,
+  };
+
+  cinderHits = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasTalent(SPELLS.CINDERSTORM_TALENT.id);
+  }
+
+  on_byPlayer_damage(event) {
+    const spellId = event.ability.guid;
+    if (spellId !== SPELLS.CINDERSTORM_DAMAGE.id) {
+      return;
+    }
+    this.cinderHits += 1;
+  }
+
+  get averageHitsPerCast() {
+    return this.cinderHits / this.abilityTracker.getAbility(SPELLS.CINDERSTORM_TALENT.id).casts;
+  }
+
+  get suggestionThreshold() {
+    return {
+      actual: this.averageHitsPerCast,
+      isLessThan: {
+        minor: 6,
+        average: 5.7,
+        major: 5.4,
```

Comment:
This seems like a really narrow window of thresholds IMO. This is saying that always getting 6 is no issue, but getting a mix of 5s and 6s could potentially be a major issue. Getting 9 perfect hits and 1 miss is a major issue.

I dunno how much you've played with Cinderstorm, but it's really hard and sometimes up to chance if you can actually hit with all 6 cinders. My opinion is this should be spaced out more.